### ISSUE TITLE
Shellcheck wrong exit code (1 instead of 0)

### ISSUE THREAD

**Issue by WaldoFR** [koalaman/shellcheck#1323]
#### For bugs
- Rule Id (if any, e.g. SC1000): /
- My shellcheck version (`shellcheck --version` or "online"): 0.4.6
- [ ] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [x] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

#!/bin/sh

return=$(shellcheck /home/informatique/system/scriptExo1.sh /home/informatique/system/voyelles.sh /home/informatique/system/inv.sh /home/informatique/system/profondeur.sh --shell=sh -f checkstyle)

echo "Return code is : $?"
```

#### Here's what shellcheck currently says:
When running this, I get the expected result (printing the result in output) and no error in errostream. However, the exit code return is 1 instead of 0.

#### Here's what I wanted or expected to see:
I think shellcheck should be exited with value 0, otherwise show what is the problem

**Comment by WaldoFR** [comment]
Ok I just read in the wiki that 1 mean "ok".
